// Copyright 2023 PingCAP, Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package DB.IncrementalSnapshotProto;

message DeltaSummary {
    uint64 persisted_applied_index = 1;
    repeated uint64 l0_ids = 2;
    uint64 ver = 3;
    bytes start = 4;
    bytes end = 5;
}

message DeltaSummariesPersisted {
    DeltaSummary summary = 1;
}